src/App.js

```diff
@@ -13,11 +13,16 @@ class App extends Component {
     this.state = {
       movies: [],
       error: '',
-      user:{}
+      user:{},
+      isLoggedIn: false,
```

Comment:
I think this approach is a great way to get our pages to render when they should.